src/background/messaging/mapRPCMessageToController.ts

```diff
@@ -57,14 +57,18 @@ export function mapRPCMessageToController<T>(
           return formatJsonRpcResultForPort(id, result);
         },
         (error: Error | DOMException | ErrorResponse) => {
-          return formatJsonRpcWalletError(
+          const normalizedError = formatJsonRpcWalletError(
             id,
             error instanceof DOMException
               ? domExceptionToError(error).message
               : 'code' in error
               ? error
               : getError(error)
           );
+          console.group('Controller error'); // eslint-disable-line no-console
```

Comment:
Do we need this in production?

Reply:
Yep, so that it's easier for users to report what happened :)